src/test/kotlin/calculator/StringCalculatorTest.kt

```diff
@@ -0,0 +1,120 @@
+package calculator
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.Arguments
+import org.junit.jupiter.params.provider.MethodSource
+import org.junit.jupiter.params.provider.ValueSource
+import java.util.stream.Stream
+
+class StringCalculatorTest {
+    private val calculator = StringCalculator()
+
+    @Test
+    fun `입력값이 null인 경우`() {
+        assertThatThrownBy {
+            calculator.calculate(null)
+        }
+            .isInstanceOf(IllegalArgumentException::class.java)
+            .hasMessage("입력값이 null 혹은 빈 값입니다.")
```

Comment:
kotlin 감수성을 추가해서 `assertThrows`를 사용해보는건 어떨까요!

```suggestion
        val assertThrows = assertThrows<IllegalArgumentException> { calculator.calculate(null) }
        assertThat(assertThrows.message).isEqualTo("입력값이 null 혹은 빈 값입니다.")

        assertThrows<IllegalArgumentException> { calculator.calculate(null) }
            .also { assertThat(it.message).isEqualTo("입력값이 null 혹은 빈 값입니다.") }
```